src/equations/acoustic_perturbation_2d.jl

```diff
@@ -0,0 +1,153 @@
+@doc raw"""
+    AcousticPerturbationEquations2D(v_mean, rho_mean, c_mean)
```

Comment:
Please mark this system of equations as experimental, such that we can still make breaking changes without having to bump the minor version number.